### ISSUE TITLE
How do you get the gpg key to stay decrypted for longer?

### ISSUE THREAD

**Issue by o-jasper** [matthiasbeyer/pass2luakit#2]
I really dont think the shortness of time is sensible, and really protects against anything.

Perhaps RTFM, but can't find it.

**Comment by matthiasbeyer** [comment]
I have gpg-agent running for this.

**Comment by o-jasper** [comment]
Right.. there is `~/.gnupg/gpg-agent.conf` where you can set `default-cache-ttl` to something else to find it.

Now, if you want to set that to a much longer time, well presumably there are higher-security keys that should have a much shorter time-to-live. Don't see how to only change the ttl for only one key id.

Doesnt matter much to me, because i have taken up [firejail](https://l3net.wordpress.com/projects/firejail/), and luakit is in its own little home folder now..
